packages/@aws-cdk/aws-ecs/lib/linux-parameters.ts

```diff
@@ -23,6 +23,20 @@ export interface LinuxParametersProps {
    * @default No shared memory.
    */
   readonly sharedMemorySize?: number;
+
+  /**
+   * The total amount of swap memory (in MiB) a container can use.
+   *
+   * @default No swap.
+   */
+  readonly maxSwap?: number;
+
+  /**
+    * This allows you to tune a container's memory swappiness behavior.
+    *
+    * @default 60
+    */
+  readonly swappiness?: number;
```

Comment:
Please include the min and max values in the description. As well as a short description of what 'swappiness' means. Feel free to lift from the ECS docs: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-linuxparameters.html

Reply:
I added some explicit descriptions for params meaning and accepted (valid) values. I also mentioned that these parameters are only supported with EC2 launch type, but I found no way to throw an error if the wrong launch type is supplied. Is there a clean and easy way to access the Service launchType attribute?